index.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const NR = require('node-resque');
+
+// SET UP THE CONNECTION
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+// DEFINE YOUR WORKER TASKS
+const jobs = {
+    start: {
+        plugins: ['jobLock', 'retry'],
+        pluginOptions: {
+            jobLock: {},
+            retry: {
+                retryLimit: 3,
+                retryDelay: (1000 * 5)
+            }
+        },
+        /* config looks like this
```

Comment:
I would imagine we would document all Javascript functions, methods, and classes; this one being no exception